src/datagrid/DataGrid.tsx

```diff
@@ -145,6 +154,22 @@ const DataGrid = ({
     );
 };
 
+// TODO - inject this to Provider
+const RowRendererComponent = memo(({ style, index, ...rest }: ViewProps & { index: number }) => {
+    const { View } = useMolecules();
```

Comment:
you will use RowRendererComponent as it is. but you'll be using useMolecules here to get a DataGridRowRenderer Component which is by default View from molecules.